templates/email/kingston/alert-update.txt

```diff
@@ -2,6 +2,10 @@ Subject: Update on report, reference RBK-[% report.id %]
 
 The following updates have been left on this report:
 
+[% IF report.category == 'Request new container' %]
```

Comment:
This has moved back below the line rather than being above, from previous review.